.gitignore

```
@@ -71,7 +71,9 @@ instance/
 .scrapy
 
 # Sphinx documentation
-doc/build/
+docs/_build/
```

```suggestion
docs/build/
```
Unless we change the CI, the sphinx documentation will still end-up in "build" and not "_build". See [here](https://github.com/pyansys/pydpf-post/blob/f9a6412f906a0e70c1a3a7fe19b88248124933d7/docs/Makefile#L9).